applications/doodle/src/doodle_exe.erl

```diff
@@ -253,7 +253,7 @@ init([Call]) ->
     process_flag('trap_exit', 'true'),
     CallId = whapps_call:call_id(Call),
     put('callid', CallId),
-    self() ! 'initialize',
+    gen_listener:cast(self(), 'initialize'),
```

Comment:
self() ! 'initialize' directs to handle_info not handle_cast, you know that right ?